### PR TITLE
Fix of comments in step-33.

### PR DESCRIPTION
**Pull request by l-korous**
This is just a tip how to fix the docs.

### CODE REVIEW

**Comment by l-korous** [comment]
Sorry, figured out this is not DG.

File: examples/step-33/step-33.cc [hunk 1]
```diff
@@ -252,12 +252,13 @@ namespace Step33
     }
 
 
-    // @sect4{EulerEquations::compute_normal_flux}
+    // @sect4{EulerEquations::numerical_normal_flux}
```
Comment (bangerth) [inline examples/step-33/step-33.cc]: This change was definitely correct.

**Comment by l-korous** [comment]
Hi,

will do, sorry for delay.

Anyway, I noticed a problem with Ubuntu compilation with Trilinos
at 2a043a2c0ad2d107c6fec37f707a94308c45c218:

/usr/bin/ld.gold: error:
/usr/local/lib/libteuchosparameterlist.a(Teuchos_ConditionXMLConverterDB.cpp.o):
requires dynamic R_X86_64_PC32 reloc against
'_ZN7Teuchos16rcp_dynamic_castIKNS_15NumberConditionIyEEKNS_18ParameterConditionEEENS_3RCPIT_EERKNS6_IT0_EEb'
which may overflow at runtime; recompile with -fPIC

There is 1000s of linker issues like this. The flag "-fPIC" fixes the
issue. I install all packages from (debian) distribution.

Best,

Lukáš Korous

On Mon, Nov 14, 2016 at 3:06 AM, Wolfgang Bangerth <notifications@github.com

**Comment by bangerth** [comment]
Lukas,

> will do, sorry for delay.

Great, thanks!


> Anyway, I noticed a problem with Ubuntu compilation with Trilinos
> at 2a043a2c0ad2d107c6fec37f707a94308c45c218:
>
> /usr/bin/ld.gold: error:
> /usr/local/lib/libteuchosparameterlist.a(Teuchos_ConditionXMLConverterDB.cpp.o):
> requires dynamic R_X86_64_PC32 reloc against
> '_ZN7Teuchos16rcp_dynamic_castIKNS_15NumberConditionIyEEKNS_18ParameterConditionEEENS_3RCPIT_EERKNS6_IT0_EEb'
> which may overflow at runtime; recompile with -fPIC
>
> There is 1000s of linker issues like this. The flag "-fPIC" fixes the
> issue. I install all packages from (debian) distribution.

Is this with the latest development version? Pull request #3044 should have 
fixed this.

Best
  W.

-- 
------------------------------------------------------------------------
Wolfgang Bangerth          email:                 bangerth@colostate.edu
                            www: http://www.math.colostate.edu/~bangerth/

**Pull request closed without merging**